test/plugin/test_filter_stdout.rb

```diff
@@ -1,84 +1,89 @@
 require_relative '../helper'
+require 'fluent/test/driver/filter'
 require 'fluent/plugin/filter_stdout'
 require 'timecop'
 require 'flexmock/test_unit'
 
 class StdoutFilterTest < Test::Unit::TestCase
-  include Fluent
   include FlexMock::TestCase
 
   def setup
     Fluent::Test.setup
+    @old_tz = ENV["TZ"]
+    ENV["TZ"] = "UTC"
     Timecop.freeze
   end
 
   def teardown
     super # FlexMock::TestCase requires this
     # http://flexmock.rubyforge.org/FlexMock/TestCase.html
     Timecop.return
+    ENV["TZ"] = @old_tz
   end
 
   CONFIG = %[
   ]
 
   def create_driver(conf = CONFIG)
-    Test::FilterTestDriver.new(StdoutFilter, 'filter.test').configure(conf)
+    Fluent::Test::Driver::Filter.new(Fluent::Plugin::StdoutFilter).configure(conf)
   end
 
-  def emit(d, msg, time)
+  def filter(d, time, record)
     d.run {
-      d.emit(msg, time)
-    }.filtered_as_array[0][2]
+      d.feed("filter.test", time, record)
+    }
+    d.filtered_records
   end
 
   def test_through_record
     d = create_driver
     time = Time.now
-    filtered = emit(d, {'test' => 'test'}, Fluent::EventTime.from_time(time))
-    assert_equal({'test' => 'test'}, filtered)
+    filtered = filter(d, Fluent::EventTime.from_time(time), {'test' => 'test'})
```

Comment:
Just `event_time()` test helper method available to get current EventTime instance instead of `time = Time.now; Fluent::EventTime.from_time(time)`.
(Many same code exist below).